backoff/fibonacci_test.go

```diff
@@ -0,0 +1,81 @@
+package backoff
+
+import (
+	"testing"
+	"time"
+)
+
+func TestFibonacci_Duration(t *testing.T) {
+	type fields struct {
+		attempt uint64
+		Jitter  bool
+		Min     time.Duration
+		Max     time.Duration
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   time.Duration
+	}{
+		{
```

Comment:
Apologies for hijacking the review; one suggestion: we could add a test for `attempt=0`, just to make sure that the returned value is 0 and not any other non-zero value.

```
{
	"must generate the correct duration for attempt=0",
	fields{
		attempt: uint64(0),
		Min:     1 * time.Minute,
		Max:     10 * time.Minute,
	},
	0,
},
```

Could add a similar test for `attempt=1`, in order to cover [this](https://github.com/deepsourcelabs/hermes/pull/3/files#diff-6ceb2d7871c325d41e01873bcc312fbb85dea5bb686b29b2ad9b6b33e0c63166R79) piece of return logic.

Reply:
One question is if attempt=0, shouldn't we return the Min instead?

Reply:
Agreed.  We do send 0 currently.  Resolving this for now.